test/zdtm/static/Makefile

```diff
@@ -216,6 +216,7 @@ TST_NOFILE	:=				\
 		pipe03				\
 		netns_lock_iptables			\
 		netns_lock_nftables			\
+		per_socket_lock_iptables		\
```

Comment:
It would be better to add a prefix to the test name, because "lock" could mean something else (e.g. file lock).
For example: `net_lock_socket_iptables`.